packages/snap/src/keyring.ts

```diff
@@ -72,9 +72,11 @@ export type ChainConfig = {
   customVerifyingPaymasterAddress?: string;
 };
 
+export type ChainConfigs = Record<string, ChainConfig>;
```

Comment:
I think we will need to change the typing of the key from number to string here in the `setConfig` method

```ts
    this.#state.config[Number(chainId)] = {
      ...this.#state.config[Number(chainId)],
      ...config,
    };

    await this.#saveState();
    return this.#state.config[Number(chainId)]!;    this.#state.config[Number(chainId)] = {
      ...this.#state.config[Number(chainId)],
      ...config,
    };

    await this.#saveState();
    return this.#state.config[Number(chainId)]!;
    
```
should be `this.#state.config[chainId.toString()]`

Reply:
The number would automatically get casted back into a string, but this is a good point to make it more explicit.